cms/templates/sudo/sudo.html

```diff
@@ -1,7 +1,8 @@
 {% block body %}
```

Comment:
This template is rendering as a partial in my browser... it's missing the core html structure of html, head, body tags. Also visually it is in nowhere land. If you are going to do this in a modal, can you have it render over a blank Studio page? Or can you do the same as you did in the LMS and render the header footer?